Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -200,13 +200,10 @@ std::optional<InteractionRegion> interactionRegionForRenderedRegion(RenderObject
     if (!isOriginalMatch && !isInlineNonBlock)
         return std::nullopt;
 
-    if (isInlineNonBlock)
-        bounds.inflate(regionRenderer.document().settings().interactionRegionInlinePadding());
-
     float borderRadius = 0;
     OptionSet<InteractionRegion::CornerMask> maskedCorners;
 
-    if (auto* renderBox = dynamicDowncast<RenderBox>(renderer)) {
+    if (auto* renderBox = dynamicDowncast<RenderBox>(regionRenderer)) {
```

Comment:
Do you think this is strictly a win? Should we be max() or min()ing them or something?

Reply:
yep, especially with the new, higher, minimum border radius